test/Container/ApplicationConfigInjectionDelegatorTest.php

```diff
@@ -83,7 +84,7 @@ public function getQueueFromApplicationPipeline(Application $app)
         return $r->getValue($pipeline);
     }
 
-    public static function assertRoute($spec, array $routes)
+    public static function assertRoute(array $spec, array $routes)
     {
         Assert::assertThat(
             array_reduce($routes, function ($found, $route) use ($spec) {
```

Comment:
I would take this to the next level and import functions:

`use function array_reduce;`

Reply:
We could do, it's not a problem, sniffs for import internal functions and constants are there, but by default these are disabled, because per discussion in other repository it looks like we don't want to have it everywhere. If now we want it's not a problem to enable these sniffs :)

Reply:
I think if we enable it, we need to document how to disable the feature per-identifier and/or per file. (I already know how to do this, but we need to ensure there's good documentation for even new contributors.)

I'd be okay with enabling this by default; it definitely forces you to consider things like file-system operations and `require`/`include` within your code!